client/src/components/pages/RobotInteractionCockpit/RobotInteractionCockpit.jsx

```diff
@@ -86,13 +108,54 @@ const RobotInteractionCockpit = (match) => {
     setParameters(currentParameters);
   };
 
+  const displayStatusIcon = (status) => {
+    if (status === 'PASS' || status === 'successful') {
+      return (
+        <CheckCircleOutlined
+          className={styles.statusIconStyle}
+          style={{ color: 'green' }}
+        />
+      );
+    }
+    if (status === 'FAIL' || status === 'failed') {
+      return (
+        <CloseCircleOutlined
+          className={styles.statusIconStyle}
+          style={{ color: 'red' }}
+        />
+      );
+    }
+    if (status === 'running') {
+      return (
+        <Loading3QuartersOutlined
+          spin
+          style={{
+            top: '40%',
```

Comment:
Please also pull this out of the code into the css module

Reply:
done